examples/specs/window_mean_difference.vl.json

```diff
@@ -34,7 +34,8 @@
           "aggregate": "average",
           "field": "AverageRating",
           "type": "quantitative"
-        }
+        },
+        "y": {"field": "Title", "type": "ordinal"}
```

Comment:
Why adding this?

Reply:
Obviously this is making it wrong

![image](https://user-images.githubusercontent.com/111269/37883160-994870d6-305d-11e8-8c3e-c90a8623d993.png)